code/app/build.gradle.kts

```diff
@@ -52,10 +52,11 @@ dependencies {
     implementation(platform(libs.firebase.bom))
     implementation(libs.firebase.firestore)
     implementation(libs.geofire.android.common)
-    implementation(libs.firebase.auth)
+   implementation(libs.firebase.auth)
 
     // Google Services
     implementation(libs.play.services.location)
+    implementation("com.google.firebase:firebase-messaging")
```

Comment:
can you move the reference to `libs.versions.toml`. Android studio can do that for you